pages/content/amp-dev/documentation/guides-and-tutorials/index.html

```diff
@@ -1,393 +1,228 @@
 ---
-$title: AMP [[ format|capitalize ]] Guides & Tutorials
+$title: Guides & Tutorials for AMP Websites
 $titles:
   navigation: Guides & Tutorials
 $view: /views/overview/guides.j2
+$path: /documentation/guides-and-tutorials/index.html
+$localization:
+  path: /{locale}/documentation/guides-and-tutorials/index.html
 $order: 1
 $hidden: true
-description: "All resources you need to create stunning [[ format if format !== 'email' else 'emails' ]] with AMP: Guides, tutorials, reference documentation and examples."
+description: 'All resources you need to build stunning websites with AMP: Guides, tutorials, reference documentation and examples.'
 flyout:
   description: Get started with AMP
   icon: guides-documents
+
+highlight_guides:
```

Comment:
Same as with `highlighted_components`, valid for all of those frontmatter data: try to only use lists of `/content/amp-dev/...` paths which are used to create actual docs via `g.doc` so you don't have to duplicate all the teaser information here manually.